### PR TITLE
ac-salesforce-integration

### PR DESCRIPTION
**Pull request by AlexCppns**

### CODE REVIEW

File: lib/external_service_new/salesforce_dispatcher.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  class Salesforce < RestClientDispatcher
+
+    attr_reader :account_id
+    def initialize(account_identifiers: nil, credential_details:, sleep_time_seconds: 5)
+      @account_id = account_identifiers.fetch("account_id")
+      @client       = Databasedotcom::Client.new(
+        client_id: credential_details.fetch("client_id"),
+        client_secret: credential_details.fetch("client_secret")
+      )
+      @client.authenticate(token: credential_details.fetch(:token), :instance_url => "http://na1.salesforce.com")
```
Comment (francois) [inline lib/external_service_new/salesforce_dispatcher.rb]: Is there an HTTPS endpoint? If so, I'd prefer to use it.

Comment (AlexCppns) [inline lib/external_service_new/salesforce_dispatcher.rb]: Looks like there is, I ll try it out.

File: wafflehouse.gemspec [hunk 2]
```diff
@@ -28,4 +28,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "bundler", "~> 1.12"
   spec.add_development_dependency "rake", "~> 10.0"
   spec.add_development_dependency "minitest", "~> 5.0"
+  spec.add_dependency 'restforce'
```
Comment (francois) [inline wafflehouse.gemspec]: Please specify some kind of version here. This would prevent us from upgrading to an incompatible version of restforce.

**Comment by francois** [comment]
LGTM

**Pull request closed without merging**